### ISSUE TITLE
provide richer info about tool calls in accessible view

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#248160]
Inspired by an investigation for MCP by @connor4312, but applicable to all - in the a11y view, we should indicate:

1) Past tool run information - in the current session and after window reload
2) Current tool run information, like the inputs that will be used
3) When there are multiple confirmation messages

**Comment by amunger** [comment]
I tried when a mcp tool confirmation box was open

pressing alt+f2 from the input box:

![Image](https://github.com/user-attachments/assets/c34572a6-e3c8-4977-bd41-1cf3704a281a)

pressing alt+f2 while focused on the item in the history:

```
I'll help you use the mcp_everything_add tool to add these numbers. [object Object]: {"a":123,"b":123123123}
Adds two numbers
```

![Image](https://github.com/user-attachments/assets/b1b2eb17-2551-4a02-b3d6-74cc09f7e2fe)

**Comment by rzhao271** [comment]
Marking as verified.

As polish, should there be a newline in this case?

![Accessible view shows answer immediately followed by tool call information on same line](https://github.com/user-attachments/assets/54e3690f-20c9-4679-8af8-868e5948d72c)

![Sighted users see a clear distinction between the tool call and the final response](https://github.com/user-attachments/assets/ab0b460d-b96e-43d1-9f35-4ad3128aa001)

**Comment by meganrogge** [comment]
At the least, there should be a space ` `.
